src/derivatives/imaging.md

```diff
@@ -306,8 +308,8 @@ A guide for using macros can be found at
     "pipeline": {
         "sub-001": {
             "anat": {
-                "sub-001_space-orig_seg-Desikan_label-GM_mask.nii.gz": "",
-                "sub-001_space-orig_seg-Desikan_label-GM_mask.json": "",
+                "sub-001_space-orig_seg-tissue_label-GM_mask.nii.gz": "",
+                "sub-001_space-orig_seg-tissue_label-GM_mask.json": "",
```

Comment:
So "_seg-" can be anything, but maybe the use of tissue is a bit confusing. Can we rather use "_seg-thalamus_label-GM" or "_seg-wholebrain_label-GM". This might be easier to understand.

Reply:
I chose "tissue" as a shorthand for "brain tissue segmentation" (which is a classic problem). But I'm not attached to it, so if wholebrain or thalamus would be preferred, I don't oppose. If we choose thalamus, perhaps label should have a nucleus name, or remove the label (`seg-thalamus_mask`).

If `seg-braintissues_label-GM` would be clearer, it could also be an option.

I am happy to change it as it is clearest.

Reply:
Okay, I have expanded this more reasonably, and gently advanced the potential combination with `atlas-`.